src/Umbraco.Web.UI.Client/src/packages/content/content-type/workspace/views/design/content-type-design-editor-tab.element.ts

```diff
@@ -87,20 +101,20 @@ export class UmbContentTypeDesignEditorTabElement extends UmbLitElement {
 						this.#groupStructureHelper.partialUpdateContainer(entry.ownerId, {
 							sortOrder: ++prevSortOrder,
 						});
-
-						i++;
 					}
+					i++;
```

Comment:
@madsrasmussen this was the problem, but it turned out that moving containers across was never implemented. this is fixed now.